src/main/java/com/spikes2212/command/drivetrains/TankDrivetrain.java

```diff
@@ -38,6 +39,10 @@ public TankDrivetrain(MotorController leftController, MotorController rightContr
         this(getClassName(DEFAULT_NAMESPACE_NAME), leftController, rightController);
     }
 
+    public TankDrivetrain(MotorController leftMaster, MotorController leftSlave, MotorController rightMaster, MotorController rightSlave) {
```

Comment:
create an additional constructor which receives 4 motor controllers and a string representing the namespace name